src/test/java/domain/player/NameTest.java

```diff
@@ -0,0 +1,46 @@
+package domain.player;
+
+import domain.player.Name;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class NameTest {
+    @Test
+    @DisplayName("문자열을 통해 이름을 생성한다.")
+    public void createName(){
+        //Given
+        String value = "도비";
+
+        //When
+        Name name = new Name(value);
+
+        //Then
+        assertEquals(name.getValue(),value);
```

Comment:
```suggestion
    public void createName() {
        // Given
        String value = "도비";

        // When
        Name name = new Name(value);

        // Then
        assertEquals(name.getValue(), value);
```
전체적으로 컨벤션이 아직 안맞춰져있어요. 컨벤션으로 리뷰걸린 클래스가 아닌 다른 클래스들도 다시 한번 확인해주세요.

[우테코 코딩 컨벤션](https://github.com/woowacourse/woowacourse-docs/tree/main/styleguide/java)도 적용부탁드려요.